tests/OTAnalytics/plugin_ui/test_cli.py

```diff
@@ -144,101 +147,105 @@ def temp_section(test_data_tmp_dir: Path, otsection_file: Path) -> YieldFixture[
     temp_otsection.unlink()
 
 
+@pytest.fixture
+def temp_otconfig(
+    test_data_tmp_dir: Path, otconfig_file: Path, ottrk_path: Path, cyclist_video: Path
+) -> YieldFixture[Path]:
+    temp_dir = test_data_tmp_dir / "temp_otconfig"
+    temp_dir.mkdir(parents=True)
+    temp_otconfig = temp_dir / otconfig_file.name
+    temp_ottrk = temp_dir / ottrk_path.name
+    temp_video = temp_dir / cyclist_video.name
+    copy2(src=otconfig_file, dst=temp_otconfig)
+    copy2(src=ottrk_path, dst=temp_ottrk)
+    copy2(src=cyclist_video, dst=temp_video)
+    yield temp_otconfig
+
+
 @pytest.fixture
 def event_list_exporter() -> EventListExporter:
     return AVAILABLE_EVENTLIST_EXPORTERS[OTC_OTEVENTS_FORMAT_NAME]
 
 
-def create_cli_args(
+@pytest.fixture
+def flow_parser() -> FlowParser:
+    return OtFlowParser()
+
+
+@pytest.fixture
+def mock_flow_parser() -> Mock:
+    parser = Mock()
+    parser.parse.return_value = ([], [])
+    return parser
+
+
+@pytest.fixture
+def video_parser() -> VideoParser:
+    return CachedVideoParser(SimpleVideoParser(OpenCvVideoReader()))
+
+
+@pytest.fixture
+def config_parser(video_parser: VideoParser, flow_parser: FlowParser) -> ConfigParser:
+    return OtConfigParser(video_parser, flow_parser)
+
+
+def create_run_config(
+    flow_parser: FlowParser,
     start_cli: bool = True,
     debug: bool = False,
+    config_file: str = CONFIG_FILE,
     track_files: list[str] | None = None,
     sections_file: str = SECTION_FILE,
+    save_dir: str = "",
     save_name: str = "",
     save_suffix: str = "",
-    event_list_exporter: EventListExporter = AVAILABLE_EVENTLIST_EXPORTERS[
-        OTC_OTEVENTS_FORMAT_NAME
-    ],
-    count_interval: int = 1,
+    event_formats: str = DEFAULT_EVENTLIST_FILE_TYPE,
+    count_intervals: int = 1,
+    # event_formats: list[str] | None = None,
```

Comment:
Do we need the commented code?

Reply:
I simply forgot to implement the feature to allow the user to pass multiple arguments to `--event-format` and `count-interval`. These were comments pertaining to this.